src/styles/styles.js

```diff
@@ -1264,12 +1264,10 @@ const styles = {
         textDecorationLine: 'none',
     },
 
-    leftPanelContainer: {
-        maxWidth: variables.sideBarWidth,
-    },
-
-    rightPanelContainer: {
-        width: variables.sideBarWidth,
+    rhpNavigatorContainer: {
```

Comment:
Abbreviations at start of name should be uppercase.
```suggestion
    RHPNavigatorContainer: {
```